### DISCUSSION TITLE
Configuration of calender ics

### DISCUSSION THREAD

**Discussion by fighttomatoes** [pretix/pretix#2324] [Ideas]
It would be nice to have the option to configurate the calender ics in pretix. My boss, for example, asks me to add an automatic notification in the ics, which alarms him 1 hour before the actual event. Also an adjustable description would be nice.
Is it possible?

**Comment by raphaelm** [comment]
We do plan introducing an adjustable description.

I do not think we will introduce notification options, as this is too specific and also is something that should be up to the end user.

**Comment by eiclu** [comment]
Is there an update to this? I am trying to import the global ics feed into our website.

It would be great to at least include the event description in the ics description. Configurable with a template would be even better.

Additionally, the event cover image could be linked as an `IMAGE` property per [RFC 7986](https://datatracker.ietf.org/doc/html/rfc7986#section-5.10)

**Reply by raphaelm** [reply]
We have made the ICS sent via email more customizable, but not he public one so far.

(Nitpick: pretix has no event cover image (yet), it only has an event header image, which is not really the same thing as just because it's a nice header, it does not mean that it is suitable for other purposes in size and content.)
